configure

```diff
@@ -130,6 +130,8 @@ DSC_SCRIPT_PATH=$DSC_SCRIPT_PATH
 
 BUILD_DEBUG=$BUILD_DEBUG
 DEBUG_FLAGS=$DEBUG_FLAGS
+# DSC cannot build with -Werror
+ENABLE_WERROR=0
```

Comment:
In the Makefile, this config.mak is included before the omi config.mak.
The omi config.mak then overwrites this back to 1. How does this fix work?